src/thread.js

```diff
@@ -38,6 +53,9 @@ class Thread {
     this._subscribe(this._address, { firstModerator: this._firstModerator, members: this._members, name: this._name })
     this._replicator.ensureConnected(this._address, true)
     const timestamp = Math.floor(new Date().getTime() / 1000) // seconds
+
+    if (this._confidential) message = this._symEncrypt(message)
```

Comment:
We likely need to stringify the message here. Not sure what happens if you try to encrypt an object.

Reply:
yeah good catch, should stringify objects somewhere, can add a test for that as well

Reply:
yep, see comment below 👍